tests/infer/test_autoguide.py

```diff
@@ -242,14 +243,28 @@ def model():
     guide()
     if auto_class is AutoLaplaceApproximation:
         guide = guide.laplace_approximation()
+    pyro.set_rng_seed(0)
+    expected = guide.call()
+    names = sorted(guide())
 
     # Ignore tracer warnings
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", category=torch.jit.TracerWarning)
         # XXX: check_trace=True fails for AutoLaplaceApproximation
         traced_guide = torch.jit.trace_module(guide, {"call": ()}, check_trace=False)
-    torch.jit.save(traced_guide, "/tmp/test_guide_serialization.pt")
-    guide_deser = torch.jit.load("/tmp/test_guide_serialization.pt")
+    f = io.BytesIO()
```

Comment:
I've switched to from `/tmp/*` files to in-memory temp buffers to avoid spurious test errors with pytest-xdist.